src/freenet/pluginmanager/PluginManager.java

```diff
@@ -820,66 +806,16 @@ public void unregisterPluginToadlet(PluginInfoWrapper pi) {
 		}
 	}
 
-	public void addToadletSymlinks(PluginInfoWrapper pi) {
```

Comment:
Actually, take a look at this method: it does not even pretend to do what it’s named for. I’m more in favor of simply removing this method before anybody notices its complete brokenness. :)